devenv/doctor.py

```diff
@@ -229,19 +229,19 @@ def main(context: Context, argv: Sequence[str] | None = None) -> int:
         if args.check_only:
             return 1
 
-    print("\nThe following problems have been identified:")
     skip: list[Check] = []
+    print("\nLet's go through the failures one by one.")
     for check in failing_checks:
-        print(f"\t❌ {check.name}".expandtabs(4))
+        print(f"❌ {check.name}".expandtabs(4))
```

Comment:
the expandtabs aren't doing anything any more

Reply:
im just gonna remove all of em it hurts my eyes